src/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/machines/AdvancedCargoOutputNode.java

```diff
@@ -206,6 +324,11 @@ public void onPlace(Player p, Block b, SlimefunItem item) {
 				BlockStorage.addBlockInfo(b, "filter-type", "whitelist");
 				BlockStorage.addBlockInfo(b, "filter-lore", "true");
 				BlockStorage.addBlockInfo(b, "filter-durability", "false");
+				if(getAttachedBlock(b).getType().equals(Material.FURNACE)) {
```

Comment:
Not a good practice, as someone could break the furnace, replace it with something else and the plugin would not take notice.